src/main/kotlin/racingcar/Application.kt

```diff
@@ -1,5 +1,125 @@
 package racingcar
 
+import camp.nextstep.edu.missionutils.Console
+import camp.nextstep.edu.missionutils.Randoms
+import kotlin.collections.all
+import kotlin.text.all
+
+var cars = mutableListOf<String>()
+var movingTime: Int = 0
+var carRunStatus = mutableMapOf<String, Int>()
+
 fun main() {
     // TODO: 프로그램 구현
+    game()
 }
+
+fun game() {
+    println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)")
+    carNameSetting()
+    println("시도할 횟수는 몇 회인가요?")
+    timeSetting()
+    println()
+    println("실행 결과")
+    stepControl()
+    winner()
+}
+
+fun input(): String {
+    return Console.readLine()
+}
+
+fun processCarNaming(): List<String> {
+    return input().split(",")
+}
+
+fun validateNamesLength(names: List<String>) {
+    if (!names.all { it.length <= 5 }) {
+        throw IllegalArgumentException("이름은 5자 이하여야 합니다.")
+    }
+    if (!names.all { it.isNotEmpty() }) {
+        throw IllegalArgumentException("이름을 비워둘 수 없습니다.")
+    }
+}
+
+fun validateNamesType(names: List<String>) {
+    for (name in names) {
+        validateNamesCharType(name)
+    }
+}
+
+private fun validateNamesCharType(name: String) {
+    if (!name.all { it.isLetter() }) {
+        throw IllegalArgumentException("이름은 알파벳만 입력 가능합니다.")
+    }
+}
+
+fun pickRandomNumber(): Int {
+    return Randoms.pickNumberInRange(0, 9)
+}
+
+fun timeSetting() {
+    val inputTime = input()
+    movingTime = inputTime.toIntOrNull()
+        ?: throw IllegalArgumentException("숫자만 가능합니다.")
+    if (movingTime <= 0) {
+        throw IllegalArgumentException("횟수는 1 이상이어야 합니다.")
+    }
+}
+
+fun stepControl() {
+    for (i in 1..movingTime) {
```

Comment:
for문 대신 repeat을 활용할 수도 있을 것 같습니다!
```suggestion
    repeat(movingTime) {
```

Reply:
우와 간편하네요. repeat...정말 좋은 거 같아요!!